src/stream_wrap.cc

```diff
@@ -273,6 +274,22 @@ void StreamWrap::SetBlocking(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+void StreamWrap::FlushSync(const FunctionCallbackInfo<Value>& args) {
+  StreamWrap* wrap = Unwrap<StreamWrap>(args.Holder());
+
+  if (!wrap->IsAlive())
+    return args.GetReturnValue().Set(UV_EINVAL);
```

Comment:
While I'm aware that SetBlocking also uses `UV_EINVAL` on this, it definitely feels like the wrong error code to be using in this case. Ah well tho.